src/NuGetPackageVerifier/already-owned-packages.txt

```diff
@@ -68,6 +80,7 @@ Microsoft.AspNetCore.Proxy
 Microsoft.AspNetCore.Razor
 Microsoft.AspNetCore.Razor.Design
 Microsoft.AspNetCore.Razor.Evolution
```

Comment:
This doesn't exist anymore. `Evolution` was renamed to `Language`. Please make sure there aren't any other non-existent packages in this list.